wfe/wfe.go

```diff
@@ -551,10 +550,10 @@ func (wfe *WebFrontEndImpl) verifyPOST(ctx context.Context, logEvent *requestEve
 		logEvent.AddError("verification of JWS with the JWK failed: %v; body: %s", err, body[:n])
 		return nil, nil, reg, probs.Malformed("JWS verification error")
 	}
+	logEvent.Payload = string(payload)
```

Comment:
I think this is still unanswered, but I'm happy to +1 on the assumption that this isn't an issue.